apps/core-angular-cli/package-lock.json

```diff
@@ -0,0 +1,12509 @@
+{
+  "name": "core-angular-cli",
```

Comment:
did we intend to include this file here?

Reply:
nope. My editor said it was an unknown file. I'll remove it.